src/main/java/com/danielagapov/spawn/Controllers/UserController.java

```diff
@@ -82,9 +86,38 @@ public ResponseEntity<Void> deleteUser(@PathVariable UUID id) {
         }
     }
 
-    // full path: /api/v1/user/friend-request
+    // full path: /api/v1/users/friend-request
     @PostMapping("friend-request")
     public FriendRequestDTO createFriendRequest(@RequestBody FriendRequestDTO friendReq) {
         return friendRequestService.saveFriendRequest(friendReq);
     }
-}
+
+    // full path: /api/v1/users/{id}/friend-requests
+    @GetMapping("{id}/friend-requests")
+    public List<FriendRequestDTO> getIncomingFriendRequests(@PathVariable UUID id) {
+        return friendRequestService.getIncomingFriendRequests(id);
+    }
+
+    // full path: /api/v1/users/events/{id}
+    @GetMapping("events/{id}")
+    public List<UserDTO> getUsersParticipatingInEvent(@PathVariable UUID id) {
+        return eventService.getParticipatingUsers(id);
+    }
+
+    // full path: /api/v1/users/{id}/friends?friendId=friendId
```

Comment:
```suggestion
    // full path: /api/v1/users/{id}/removeFriend?friendId=friendId
```
More descriptive endpoint url. Generally, you should use verbs like this to signify what the endpoint does